src/pkg/packaging/rpm/package.targets

```diff
@@ -271,6 +281,86 @@
           UseHardlinksIfPossible="False" />
 
   </Target>
+  
+  <Target Name="GenerateRuntimeDependenciesRpm">
+    <PropertyGroup>
+      <RpmPackageName>$(RuntimeDependenciesRpmPkgName)</RpmPackageName>
+      <RpmPackageVersion>$(SharedFrameworkNugetVersion)</RpmPackageVersion>
+      <RpmFile>$(DotnetRuntimeDependenciesPackageInstallerFile)</RpmFile>
+      <ConfigJsonName>dotnet-runtime-deps-rpm_config_$(OutputRid).json</ConfigJsonName>
+      <ConfigJsonFile>$(rpmPackagingConfigPath)$(ConfigJsonName)</ConfigJsonFile>
+      <RpmIntermediatesDir>$(PackagesIntermediateDir)$(RpmPackageName)/$(RpmPackageVersion)</RpmIntermediatesDir>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <rpmLayoutDirectory>$(RpmIntermediatesDir)/rpmLayoutDirectory/</rpmLayoutDirectory>
```

Comment:
(nit) why the difference in casing here?  Above we say `RpmPackageName`, but then here `rpmLayoutDirectory`.